packages/react/src/link.jsx

```diff
@@ -130,10 +126,10 @@ const navLinkProps = (
     activeClassName = '',
     ariaCurrent = 'true',
 
-    rudy,
+    api,
   } = props
 
-  const { getLocation, options, routes } = rudy
+  const { getLocation, options, routes } = api
```

Comment:
`option` and `routes` contain unserialisable things like functions (e.g. `defaultParams`), which is why I went with the context method rather than storing the required stuff in redux.